Simplenote/Classes/InterlinkResultsController.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import SimplenoteFoundation
+
+
+// MARK: - InterlinkResultsController
+//
+class InterlinkResultsController {
+
+    /// ResultsController: In charge of CoreData Queries!
+    ///
+    private let resultsController: ResultsController<Note>
+
+    /// Limits the maximum number of results to fetch
+    ///
+    var maximumNumberOfResults = Settings.defaultMaximumResults
+
+
+    /// Designated Initializer
+    ///
+    init(viewContext: NSManagedObjectContext) {
+        resultsController = ResultsController<Note>(viewContext: viewContext, sortedBy: [
+            NSSortDescriptor(keyPath: \Note.content, ascending: true)
+        ])
+
+        resultsController.predicate = NSPredicate.predicateForNotes(deleted: false)
+        try? resultsController.performFetch()
+    }
+
+    /// Returns the collection of Notes filtered by the specified Keyword in their title, excluding a specific ObjectID
+    /// - Important: Returns `nil` when there are no results!
+    ///
+    func searchNotes(byTitleKeyword keyword: String, excluding excludedID: NSManagedObjectID?) -> [Note]? {
+        filter(notes: resultsController.fetchedObjects, byTitleKeyword: keyword, excluding: excludedID)
+    }
```

Comment:
Does it search with only one keyword? I though maybe it's nice to have a behaviour similar to the main search: split by words and search for all of them.

Reply:
You're **SO** right!! updating shortly!! I can't believe I missed that. Thank you for pointing it out!!